### PR TITLE
Update 1.2 Seth Documentation

### PR DESCRIPTION
**Pull request by Caleb-Hill**
Updates to the seth directory markdown files.

### CODE REVIEW

File: docs/seth/0.2/contracts.md [hunk 1]
```diff
@@ -41,7 +43,7 @@ You can also use the copy already existing in the repo:
     cd sawtooth-seth/
     cp contracts/examples/simple_intkey/simple_intkey.sol contracts/contract.sol
 
-# Compiling Contracts
+## Compiling Contracts {#compiling-contracts}
```
Comment (agunde406) [inline docs/seth/0.2/contracts.md:46]: The getting started link on line 67 should be fixed

File: docs/seth/0.2/permissions.md [hunk 2]
```diff
@@ -2,6 +2,8 @@
 title: Permissions
 ---
 
+# Permissions
+
```
Comment (agunde406) [inline docs/seth/0.2/permissions.md:6]: the link to the introduction below should be updated

File: docs/seth/0.2/dapps.md [hunk 3]
```diff
@@ -25,11 +27,11 @@ programatically interact with Seth. Sawtooth provides a general-purpose
...
 transactions that could be used for this. However, this would require a
 complete understanding of the
-`Seth Transaction Family Spec <seth_transaction_family_spec>`{.interpreted-text
-role="doc"}. Instead, Seth comes with a web server that implements much
+[Seth Transaction Family Spec](./seth_transaction_family_spec.html).
+Instead, Seth comes with a web server that implements much
 of the [Ethereum JSON-RPC
 API](https://github.com/ethereum/wiki/wiki/JSON-RPC), the
-`seth-rpc <seth-rpc-reference-label>`{.interpreted-text role="ref"}
+(seth-rpc)[./cli_reference.html#seth-rpc-reference-label]
```
Comment (agunde406) [inline docs/seth/0.2/dapps.md:34]: Formatting switch here ()[] instead of []()

File: docs/seth/0.2/seth_transaction_family_spec.md [hunk 4]
```diff
@@ -296,7 +298,7 @@ the spec, but will be used in later versions:
     system or account balances, transferring value between accounts is
     not meaningful.
 
-# Execution
+## Execution {#execution}
```
Comment (agunde406) [inline docs/seth/0.2/seth_transaction_family_spec.md:301]: There is some formatting issues with the lists in this section.

**Review by agunde406 — CHANGES_REQUESTED** [review]

**Review by vaporos — APPROVED** [review]

**Comment by agunde406** [comment]
This documentation will not be ported to the website.

**Pull request closed without merging**